main.go

```diff
@@ -99,10 +101,15 @@ func generate(efs *embedded.EFS, searchDir, mainAPIFile, destDir, baseName, mark
 		"swagger-ui-bundle.js",
 		"swagger-ui-standalone-preset.js",
 	} {
-		data, exists := fs.ContentAsBytes(name)
-		if !exists {
+
+		file, err := subFS.Open(name)
+		if err != nil {
 			return errs.Newf("unable to locate %s", name)
 		}
+		data, err := ioutil.ReadAll(file)
+		if err != nil {
+			return errs.Newf("unable to read %s", name)
+		}
```

Comment:
Rather than all this, consider using `fs.ReadFile()` instead. Also eliminates the issue that you aren't closing the file you opened in this loop.

Reply:
Ha, good catch! I'll update.